third_party_license_file_generator/site_packages.py

```diff
@@ -241,7 +255,7 @@ def _read_all_module_metadatas_and_license_files(self):
                 self._module_licenses_by_module_name[module_name] = license_file
 
     def _read_license(self, license_path):
```

Comment:
This fails for me with Python2.7 (I know, I know, nobody should use it!)- I believe it can be worked around with the following import line:

`from codecs import open`

This will replace the open method with one that works in the desired way (for both Python2.7 and Python3.x)